src/react/components/Tabs.scss

```diff
@@ -8,7 +8,7 @@
 .Tab {
   display: inline-block;
   margin-bottom: -1px;
-  padding: 5px 15px;
+  padding: 2px 20px;
   color: #2C84BC;
```

Comment:
@developish  why not `$color-text-link` here?
I see a handful of references to the #2C84BC throughout the codebase... should those get changed to `$color-text-link` too?
![Screen Shot 2019-09-23 at 1 20 59 PM](https://user-images.githubusercontent.com/12177541/65455704-03b66a80-de05-11e9-8dfe-5c92520f5ecc.png)